messages/teleporter/message_handler.go

```diff
@@ -167,7 +183,47 @@ func (m *messageHandler) ShouldSendMessage(destinationClient vms.DestinationClie
 		return false, nil
 	}
 
-	return true, nil
+	var decision bool = true
+	deciderClientValue := reflect.ValueOf(m.deciderClient)
+	if deciderClientValue.IsValid() && !deciderClientValue.IsNil() {
```

Comment:
Can we not just check `if m.deciderClient != nil`?

Reply:
It's an interface, not a regular pointer. I googled and found this seemed to be the "right" way. Certainly open to suggestions though.